torch/csrc/utils/tensor_numpy.cpp

```diff
@@ -371,7 +371,7 @@ at::Tensor tensor_from_cuda_array_interface(PyObject* obj) {
   {
     PyObject *py_strides = PyDict_GetItemString(cuda_dict, "strides");
     if (py_strides != nullptr && py_strides != Py_None) {
-      if (PySequence_Length(py_strides) == -1 || PySequence_Length(py_strides) != sizes.size()) {
+      if (PySequence_Length(py_strides) == -1 || PySequence_Length(py_strides) != static_cast<Py_ssize_t>(sizes.size())) {
```

Comment:
Don't we prefer to cast the result of `PySequence_Length` to `size_t`?
From a quick search, looks like `Py_ssize_t` is a `int` only.